.pre-commit-config.yaml

```diff
@@ -0,0 +1,8 @@
+repos:
```

Comment:
Shouldn't we also add config for CI stuff like [here](https://pre-commit.ci/#configuration) to setup commit msg etc.

Reply:
I think that isn't required, it uses a default commit message which will make sense.